cpp/daal/include/algorithms/classifier/binary_confusion_matrix_types.h

```diff
@@ -115,6 +115,8 @@ namespace interface1
 struct DAAL_EXPORT Parameter : public daal::algorithms::Parameter
 {
     Parameter(double beta = 1.0);
+    Parameter(const Parameter & other);
```

Comment:
Shouldn't these have a definition somewhere for the method being declared?

Reply:
Here is the definition:
https://github.com/uxlfoundation/oneDAL/pull/3001/files#diff-1f596df9c7acc13d3569a06d85958935a731aeb75e8a97575942d9aa2997306eR46